src/Common/EnvironmentVariableNames.cs

```diff
@@ -1,10 +1,9 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#pragma warning disable IDE0240 // Nullable directive is redundant (when file is included to a project that already enables nullable
-
+#pragma warning disable IDE0240
 #nullable enable
```

Comment:
ditto, should we remove the `#nullable enable` an

Reply:
If you see changes like this, it is because I initially tried to remove it, but then added it back again because it is a shared source file and cannot be removed.

Reply:
I've added the comment back to reduce changes in this PR. Keep in mind, this is a shared source file.
![image](https://github.com/user-attachments/assets/38b03bf4-918e-4b7f-ba98-b3a27c918a6b)